lib/vsc/utils/fancylogger.py

```diff
@@ -562,6 +654,35 @@ def _logToSomething(handlerclass, handleropts, loggeroption, enable=True, name=N
     return handler
 
 
+def _screenLogFormatterFactory(colorize=Colorize.NEVER, stream=sys.stdout):
```

Comment:
add unittest for this function, we'll add the required modules to the test enviroment